### DISCUSSION TITLE
nwg-panel: missing icon picker

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#390] [General]
@Antiz96 - would you mind making nwg-icon-picker a hard nwg-panel dependency, instead of optional? I've just discovered its absence after default installation.

**Comment by Antiz96** [comment]
@nwg-piotr :wave: 

I think making nwg-icon-picker a hard dependency for nwg-panel would be a bit unexpected, as it is in fact optional (e.g. I've been personally using nwg-panel for months without having nwg-icon-picker installed).
However, I think it would make sense to make it a hard dependency of nwg-**shell**, (which is where it is actually missing in my opinion) as someone installing this metapkg probably expects to get the whole nwg suite.  
How does that sounds?

**Reply by nwg-piotr** [reply]
Very well. I only need it to be installed by default either way.

**Reply by Antiz96** [reply]
Done: https://gitlab.archlinux.org/archlinux/packaging/packages/nwg-shell/-/commit/59f316897300b6eb31d2ef89f4ca1bf71edd95b7 :wink:

**Reply by nwg-piotr** [reply]
Many thanks! I'll make a fresh nwg-iso build on this occasion. :)
